src/arkode/arkode_sprkstep.c

```diff
@@ -0,0 +1,781 @@
+/*---------------------------------------------------------------
+ * Programmer(s): Cody J. Balos @ LLNL
+ *---------------------------------------------------------------
+ * SUNDIALS Copyright Start
+ * Copyright (c) 2002-2022, Lawrence Livermore National Security
+ * and Southern Methodist University.
+ * All rights reserved.
+ *
+ * See the top-level LICENSE and NOTICE files for details.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ * SUNDIALS Copyright End
+ *---------------------------------------------------------------
+ * This is the implementation file for ARKODE's SPRK time stepper
+ * module.
+ *--------------------------------------------------------------*/
+
+#include "arkode/arkode_sprkstep.h"
+
+#include <arkode/arkode.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sundials/sundials_math.h>
+#include <sundials/sundials_nvector.h>
+#include <sundials/sundials_types.h>
+#include <sunnonlinsol/sunnonlinsol_newton.h>
```

Comment:
```suggestion
```

Reply:
It looks like this file still includes `<sunnonlinsol/sunnoninsol_newton.h>`, but I don't see why that is ever needed here.